spell/spell-reviewer-goerli-checklist.md

```diff
@@ -96,6 +93,7 @@ Spell Actions:
     * [ ] `tau` parameter used is the old `tau` value
   * [ ] Autoline (`line`) + Liquidation Oracle Price Bump (`val`)
     * [ ] Enable Autoline
+      * [ ] `ilk` follows format "RWAXXX-A"
```

Comment:
Not the case for D3M fyi

Reply:
what you mean? It should refer to RWA only